accounts/tasks.py

```diff
@@ -38,7 +38,7 @@ def update_portfolio_value():
         portfolio_value = user.current_portfolio_value()
         if user.portfolio_value != portfolio_value:
             user.portfolio_value = portfolio_value
-            user.save(update_fields=['portfolio_value'])
+            user.save()
```

Comment:
OK, There is iterator used. It may works. I thought that whole users list is fetched at once, but iterator is OK.